Shared/App/AppDelegate.swift

```diff
@@ -1,13 +1,17 @@
 
 import UIKit
 import ContentsquareModule
+import ContentsquareErrorAnalysisModule
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        Contentsquare.logLevel = .info        
+        Contentsquare.logLevel = .info
+        ErrorAnalysis.setURLMaskingPatterns([
+            "https://httpstat.us/:status_code/person/:person_id/store/:store_id"
+        ])
```

Comment:
Is this supposed to come from config for real scenarios?

Reply:
no, it should work like this for real scenarios.

Reply:
Got it. Thanks for clarifying.